MapboxDirections/MBDirections.swift

```diff
@@ -107,8 +107,6 @@ open class Directions: NSObject {
     
     public typealias MatchCompletionHandler = (_ waypoints: [Tracepoint]?, _ routes: [Match]?, _ error: NSError?) -> Void
```

Comment:
These typealiases need documentation comments. Otherwise the jazzy documentation won’t show their underlying signatures.